site/_posts/2023-03-10-growing-is-hard.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog-post
+title: Building a team is hard&#58; part I
```

Comment:
Is part 2 about the technology and domain? because those are worth addressing (our particular tech stack, java in general is not as sexy, the auth domain has unique challenges...)

Reply:
not sure what part 2 is yet.. but figured it would be something. :-)